Userland/Libraries/LibCards/CardPainter.cpp

```diff
@@ -467,4 +485,18 @@ void CardPainter::paint_highlighted_card(Gfx::Bitmap& bitmap, Gfx::Bitmap const&
     painter.blit({ 4, 4 }, source_to_highlight, source_to_highlight.rect().shrunken(8, 8));
 }
 
+void CardPainter::paint_disabled_card(Gfx::Bitmap& bitmap, Gfx::Bitmap const& source_to_disable)
+{
+    Gfx::Painter painter { bitmap };
+    auto paint_rect = source_to_disable.rect();
+    auto disabled_color = Color(Color::Black);
+    disabled_color.set_alpha(s_disabled_alpha);
+
+    painter.fill_rect_with_rounded_corners(paint_rect, Color::Black, Card::card_radius);
+    paint_rect.shrink(2, 2);
+    painter.fill_rect_with_rounded_corners(paint_rect, Color::White, Card::card_radius - 1);
+    painter.blit({ 0, 0 }, source_to_disable, source_to_disable.rect());
+    painter.fill_rect_with_rounded_corners(paint_rect, disabled_color, Card::card_radius - 1);
```

Comment:
Painting the rectangles feels excessive since the source bitmap will already have those. You should be able to do this with `painter.blit_filtered()` as with `paint_inverted_card()`.